lib/platform-stats.js

```diff
@@ -67,7 +67,7 @@ export const updateStationsAndParticipants = async (
     }
 
     stationStats.total++
-    if (m.fraudAssessment === 'OK') stationStats.accepted++
+    if (m.taskingEvaluation === 'OK' && m.majorityEvaluation === 'OK') stationStats.accepted++
```

Comment:
We might want to refactor this and make it safer by adding a helper method `m#accepted()` or `m#ok()`. Wdyt?

Reply:
I'd rather not hide these two conditions in a helper function - we had that before with `fraudAssessment`, and it caused the introduction of subtle bugs.

The definition of what is an accepted measurement is different in different places.

We could use something like "m#rewarded", but again, I am concerned that we will tend to use it incorrectly in places that should treat non-majority measurements as accepted.